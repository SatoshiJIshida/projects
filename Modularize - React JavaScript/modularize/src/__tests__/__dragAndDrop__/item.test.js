import React from "react";
import Item from "../../components/dragAndDrop/item";

// Library Jest taken from source https://jestjs.io/

describe("Draggable items", () => {
  it("should be defined", () => {
    expect(Item).toBeDefined();
  });

  it("should render correctly", () => {
    const item = <Item name="item-test" />;
    expect(item).toMatchSnapshot();
  });
});
