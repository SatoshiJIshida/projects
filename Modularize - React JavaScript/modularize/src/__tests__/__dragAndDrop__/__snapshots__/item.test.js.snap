// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Draggable items should render correctly 1`] = `
<Item
  name="item-test"
/>
`;
